### PR TITLE
Meeting

### PR DESCRIPTION
**Pull request by HeloiseFerla**

### CODE REVIEW

**Review by Othmane-Khiter — DISMISSED** [review]
it seems ok

File: src/pages/Meeting/Room/AlreadyBooked/index.jsx [hunk 1]
```diff
@@ -1,28 +1,37 @@
...
+import 'moment/locale/fr';
 import Avatar from '@mui/material/Avatar';
 import Badge from '@mui/material/Badge';
-import avatarManon from 'assets/Img/users/e1.png';
-import avatarPauline from 'assets/Img/users/e2.png';
-import avatarJules from 'assets/Img/users/e3.png';
-import avatarSamuel from 'assets/Img/users/e4.png';
-import avatarAlexandre from 'assets/Img/users/e5.png';
-import avatarMaxime from 'assets/Img/users/e6.png';
+// import avatarMaxime from 'assets/Img/users/e6.png';
```
Comment (Hacene010) [inline src/pages/Meeting/Room/AlreadyBooked/index.jsx]: Dead code !!!

File: src/pages/Meeting/Room/AlreadyBooked/index.jsx [hunk 2]
```diff
@@ -1,28 +1,37 @@
...
   reservation,
   setAlreadyBooked,
   setValidation,
 }) {
+  const [participants, setParticipants] = useState([]);
+
+  useEffect(() => {
+    const { id } = reservation;
+    axios
+      .get(`http://localhost:5000/meeting/${id}/participants`)
```
Comment (Hacene010) [inline src/pages/Meeting/Room/AlreadyBooked/index.jsx]: don't forget to put your url into a .env file

File: src/pages/Meeting/Room/Share/index.jsx [hunk 3]
```diff
@@ -17,10 +13,14 @@ export default function Share({ reservation, setValidation, setShare }) {
   };
   return (
     <SShare>
-      <p>Vous venez de reserver la salle de réunion n° {reservation.roomId}</p>
+      {console.log(reservation)}
```
Comment (Hacene010) [inline src/pages/Meeting/Room/Share/index.jsx]: Console.log :-/

File: src/pages/Meeting/Room/Validation/ValidationPopup/index.jsx [hunk 4]
```diff
@@ -42,16 +60,48 @@ ValidationPopup.propTypes = {
...
     roomId: propTypes.string,
-    day: propTypes.string,
     slot: propTypes.string,
+    userFirstname: propTypes.string,
+    userLastname: propTypes.string,
+    userPicture: propTypes.string,
+    roomPicture: propTypes.string,
   }),
   setShare: propTypes.func,
+  // user: propTypes.shape({
```
Comment (Hacene010) [inline src/pages/Meeting/Room/Validation/ValidationPopup/index.jsx]: Dead code

**Review by Hacene010 — CHANGES_REQUESTED** [review]
Saw biiig evils :-/

**Review by Othmane-Khiter — DISMISSED** [review]
ok

**Review by Hacene010 — DISMISSED** [review]

**Review by Hacene010 — APPROVED** [review]

**Review by Othmane-Khiter — APPROVED** [review]
it's ok for me

**Pull request merged**